spec/diddocuments.md

```diff
@@ -15,16 +13,10 @@ is an approach similar to multibase, making them self-describing and terse.
 
 ### DID Document from KERI Events
 The [[ref: KERI event stream]] represents a cryptographic chain
-of custody from the [[ref: AID]] itself down to the current set of signing keys and next rotation key commitments,
-as well as other data such as service endpoints.  When generating a
-DID document an implementation must "walk the [[ref: KEL]]" (follow the chain of key events) to determine the current key state.  Due to the fact that the KERI protocol
-allows for non-establishment event (events that don't change key state) to be intermixed in a [[ref: KERI event stream]] with establishment
-events (those that change key state), the last event may not be an establishment event and thus can not be relied on to
-provide key information.  In addition, the current set of witnesses is calculated from the initial set declared in the
-inception event and "adds" and "cuts" declared in rotation events.
+of custody from the [[ref: AID]] itself down to the current set of keys and the cryptographic commitment to the next rotation key(s). The [[ref: KERI event stream]] also contains events that do not alter the [[ref: AID]] key state, but are useful for the DID document, such as the supported domains, current set of service endpoints, etc. A did:webs resolver produces the DID document by processing the [[ref: KERI event stream]] to determine the current key state. We detail the different events in (KERI event details)[#KERI-event-details] below and show how they change the DID Document. The mapping from [[ref: KERI event stream]] to the properties of the DID Document is the core of the did:webs resolver logic. There are many ways to receive the [[ref: KERI event stream]], but only one way to process a [[ref: KERI event stream]] to produce the verifiable DID document. Understanding the optimal way to retrieve the [[ref: KERI event stream]] is beyond the scope of the spec, but a reference implementation of the resolver that details these techniques is being developed alongside this spec. The important concepts are that the entire [[ref: KERI event stream]] is used to produce and verify the DID document.
```

Comment:
I didn’t get a chance to review this before it was merged but still have comments.

Why the line "There are many ways to receive the [[ref: KERI event stream]]”?  AFAIK, the way in did:webs to retrieve the KERI event stream is to GET the file.  KERI may have other ways (and perhaps we can do a side-reference to that effect), but in did:webs, there is only one way.

Is that not right?

Reply:
Sorry, I merged because I thought the PR was largely fixing typos, not doing something substantive.

Since the KERI event stream can be fetched by GETting the file from any of the DID locations (not necessarily the one where the DID doc was fetched), the statement about "many ways" could be referring to did:webs synonyms rather than non-did:webs KERI features. But that not have been the intent.

Reply:
I created some confusion sorry about that @swcurran @dhh1128, the initial commits were a pass to cleanup prior to adding content, but then I added new content since i hadn't seen any reviews. Soon after, there was some significant activity (Markus/Stephen/Daniel all added input). I will update the PR title to reflect the more significant content.

Reply:
Since this has been merged (sorry for that), instead of reverting I will do a follow-on PR based on our discussions here.

Reply:
> I didn’t get a chance to review this before it was merged but still have comments.
> 
> Why the line "There are many ways to receive the [[ref: KERI event stream]]”? AFAIK, the way in did:webs to retrieve the KERI event stream is to GET the file. KERI may have other ways (and perhaps we can do a side-reference to that effect), but in did:webs, there is only one way.
> 
> Is that not right?

@swcurran I think the spec is migrating away from explicityly stating 'files' and toward 'KERI event stream' resources that is retrieved via a GET on the .keri for the did:webs DID..... that 'KERI event stream' could come from file(s), database(s), etc. and could be utilizing all kinds of interesting infrastructure. To address this and your related cache issue [I give more details here](https://github.com/trustoverip/tswg-did-method-webs-specification/issues/43#issuecomment-1736429577)
Happy to discuss, I appreciate the review and think we are getting close to the right level of abstraction for the spec.